crates/generate/src/types/mod.rs

```diff
@@ -22,12 +22,9 @@ pub fn define_datatype(names: &Names, namedtype: &witx::NamedType) -> TokenStrea
             witx::Type::Union(u) => union::define_union(names, &namedtype.name, &u),
             witx::Type::Handle(h) => handle::define_handle(names, &namedtype.name, &h),
             witx::Type::Builtin(b) => define_builtin(names, &namedtype.name, *b),
-            witx::Type::Pointer(p) => define_witx_pointer(
-                names,
-                &namedtype.name,
-                quote!(wiggle_runtime::GuestPtrMut),
-                p,
-            ),
+            witx::Type::Pointer(p) => {
+                define_witx_pointer(names, &namedtype.name, quote!(wiggle_runtime::GuestPtr), p)
+            }
             witx::Type::ConstPointer(p) => {
                 define_witx_pointer(names, &namedtype.name, quote!(wiggle_runtime::GuestPtr), p)
             }
```

Comment:
Since we've unified the concept of pointers, could we not match both in one arm instead of having two separate arms doing exactly the same thing?